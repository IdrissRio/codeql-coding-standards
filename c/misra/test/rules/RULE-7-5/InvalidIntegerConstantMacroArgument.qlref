rules/RULE-7-5/InvalidIntegerConstantMacroArgument.ql