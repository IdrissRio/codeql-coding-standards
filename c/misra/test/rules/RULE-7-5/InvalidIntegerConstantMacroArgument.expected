| test.c:48:13:48:17 | ... + ... | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:49:13:49:18 | access to array | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:50:13:50:19 | access to array | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:51:5:51:22 | 255 | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:52:5:52:17 | 1 | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:53:5:53:18 | 0 | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:54:5:54:17 | 0 | Argument to integer constant macro UINT8_C must be an integer literal. |
| test.c:55:5:55:20 | 0 | Argument to integer constant macro UINT8_C must be an integer literal. |
